### DISCUSSION TITLE
database read only mode

### DISCUSSION THREAD

**Discussion by cezar-radan** [codenotary/immudb#1392] [Q&A] [answered]
There is a way to set/ configure immudb database to work in "read only" mode. 
I don't want, for a while, that new entries (key-value) to be created inside a database.  

I would like to perform some maintenance tasks and to reconcile/to check some data, stored in my database, and for me would be perfect if new data will not be added inside database.

**Comment by jeroiraz** [comment, marked as answer]
it's a good point, currently the grpc server does not expose a read-only mode. However, if a database is configured to be a replica the only method to write it's using replication.
Would you mind opening a feature request to support readonly mode @cezar-radan ?

**Reply by cezar-radan** [reply]
Sure.
I did that -> https://github.com/codenotary/immudb/issues/1419
Thank you for considering my request!
